pkg/api/ops/ops.go

```diff
@@ -46,6 +64,30 @@ func (op OpCode) String() string {
 		14:  "GenericTracepoint",
 		23:  "Clone",
 		24:  "Data",
+		25:  "Cgroup",
 		254: "Test",
 	}[op]
 }
+
+type CgroupOpCode int
```

Comment:
It's better to have the enumeration typed. E.g.,:

```
const (
      MSG_OP_CGROUP_UNDEF CgroupOpCode = iota
      MSG_OP_CGROUP_MKDIR CgroupOpCode
)
```

Reply:
Done.